src/styles/theme/helper.scss

```diff
@@ -44,3 +44,7 @@ $directions: 'top', 'right', 'bottom', 'left';
   color: $global-inverse-color !important;
   fill: $global-inverse-color !important;
 }
+
+.oc-height-medium {
+  height: $height-medium !important;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5880368)